### PR TITLE
Added Limit Mode and edited UGDG

### PR DESCRIPTION
**Pull request by Weng-Kexin**
### new classes:
- Limit
- LimitList
- LimitUi

### added dolla.ui package to store all Ui related classes! 

*limit mode not functional yet!!

### CODE REVIEW

**Review by yetong1895 — APPROVED** [review]

File: src/main/java/dolla/Ui.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
             + "| |_| | |_| |   <  __/\n"
             + "|____/ \\__,_|_|\\_\\___|\n";
 
-    private static String line = "\t____________________________________________________________";
+    public static String line = "\t____________________________________________________________";
```
Comment (omupenguin) [inline src/main/java/dolla/Ui.java:22]: Why the change to public?

**Review by omupenguin — APPROVED** [review]

File: src/main/java/dolla/Ui.java (hunk 1 above)
Comment (Weng-Kexin) [reply src/main/java/dolla/Ui.java:22]: wanna access it from LimitUi class (might be able to change back to private after moving all of them to the same package AKA dolla.Ui)

**Review by tatayu — APPROVED** [review]
